### PR TITLE
Bump kubekins to v20180417-006059177

### PR DESCRIPTION
**Pull request by krzyzacy**
takes in https://github.com/kubernetes/test-infra/pull/7710

/hold
/assign @BenTheElder @shyamjvs

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -1272,6 +1272,7 @@
     "args": [
       "--check-leaked-resources",
       "--cluster=test-6724261826",
+      "--env=CLOUDSDK_CORE_PRINT_UNHANDLED_TRACEBACKS=1",
```
Comment (krzyzacy) [inline jobs/config.json]: wait... where is this coming from :-\

Comment (krzyzacy) [reply jobs/config.json]: ah - https://github.com/kubernetes/test-infra/pull/7700

**Comment by wojtek-t** [comment]
@krzyzacy - I don't fully understand it. Why it contains copy of https://github.com/kubernetes/test-infra/pull/7700 ?

**Comment by krzyzacy** [comment]
@wojtek-t you need to remove https://github.com/kubernetes/test-infra/pull/7716/files#diff-d060ec1854d131bd0d448821263f9400L1034

if you look at the combined changed files, those lines will be gone

**Comment by krzyzacy** [comment]
ack

**Comment by krzyzacy** [comment]
canary looks fine, kops quota is 'fixed' (for now, we have https://github.com/kubernetes/test-infra/pull/7740 open), we can get this in and then bump @Q-Lee's change

/hold cancel
👀

**Pull request merged**